src/utils/database-url.ts

```diff
@@ -0,0 +1,31 @@
+import type { DbConfigInterface } from '../interfaces/project-create-interface';
+
+export default function buildDatabaseUrl(config: DbConfigInterface) {
+  let connectionString;
```

Comment:
implicit any, nothing is typed here.

Can you set the return type of the function and type the `connectionString` variable?

Reply:
👍